src/components/Todo.js

```diff
@@ -0,0 +1,43 @@
+const TodoList = () => {
+  const data = [
+    {
+      id: 1,
+      title: 'Setup React website',
+      author: 'Zunair',
+    },
+    {
+      id: 2,
+      title: 'Space and the world',
+      author: 'Ali',
+    },
+    {
+      id: 3,
+      title: 'Science fiction',
+      author: 'Akram ',
+    },
+  ];
+  return (
+    <>
+
+      <div className="container">
+        {data.map((item) => (
+          <div key={item.id}>
+            <h2>{item.title}</h2>
+            <p>
+              By
+              {item.author}
+            </p>
+            <button type="button" id="remove">Remove</button>
+          </div>
```

Comment:
- [ ] according to project requirement, kindly make this a component that takes props and import here